inlineplz/util/git.py

```diff
@@ -13,7 +13,7 @@ def current_sha():
 def diff(start, end):
     return subprocess.check_output(
         ['git', 'diff', '-M', start + '..' + end]
-    ).strip().decode('utf-8', errors='replace')
+    ).decode('utf-8', errors='replace')
```

Comment:
this is the fix, the rest of the PR is just convenience

Reply:
👍